src/CTA.Rules.Models/Actions/Packageaction.cs

```diff
@@ -7,6 +7,7 @@ namespace CTA.Rules.Models
     public class PackageAction
     {
         public string Name { get; set; }
+        public string PreviousVersion { get; set; }
```

Comment:
nit: consider renaming to `OriginalVersion` to match the property name in `UpgradePackageMetric` and also reduce ambiguity as to what the value represents (i.e. `PreviousVersion` could also mean "the version released before `PackageVersion`", but `OriginalVersion` has fewer interpretations IMHO).